test/org/javarosa/test/utils/FormParserHelper.java

```diff
@@ -0,0 +1,47 @@
+package org.javarosa.test.utils;
```

Comment:
I like this change, but is there a reason you moved the classes out of the xform.parse package? I think they belong there, rather than in a new “utils” package.

Reply:
@dcbriccetti Since this is code is now used by tests outside of the`xform.parse` package, I wanted to put it in a common place, not related to any particular feature of the production code (like xform parsing).

Reply:
I’m not sure I understand. If the classes are related to parsing, it makes sense to have them in a `parse` package, even if they are used from classes outside the package.

Reply:
@dcbriccetti Hmm I just treated this class as a helper class and wanted to keep it outside of packages related to the actual production code. On second thought, it actually tests the form parsing codebase so it *is* related to the production code. I'll move it back to the `xform.parse` package. Thanks!

Reply:
@dcbriccetti Done!